Unable to compile

A mismatched types error from `fello` dependency at least commit `a8c0686`
```
error[E0308]: mismatched types
  --> /Users/hanhaoshen/.cargo/git/checkouts/fount-d923d0582b4b6119/a8c0686/fello/src/meta/attributes.rs:38:31
   |
38 |             if fs_selection & FS_SELECTION_ITALIC != 0 {
   |                ------------   ^^^^^^^^^^^^^^^^^^^ expected struct `SelectionFlags`, found `u16`
   |                |
   |                expected because this is `SelectionFlags`


error[E0308]: mismatched types
  --> /Users/hanhaoshen/.cargo/git/checkouts/fount-d923d0582b4b6119/a8c0686/fello/src/meta/attributes.rs:50:35
   |
50 |             if head.mac_style() & MAC_STYLE_BOLD != 0 {
   |                ----------------   ^^^^^^^^^^^^^^ expected struct `MacStyle`, found `u16`
   |                |
   |                expected because this is `MacStyle`


error[E0308]: mismatched types
   --> /Users/hanhaoshen/.cargo/git/checkouts/fount-d923d0582b4b6119/a8c0686/fello/src/meta/metrics.rs:153:57
    |
153 |             if os2.fs_selection() & USE_TYPO_METRICS != 0 {
    |                -------------------------------------    ^ expected struct `SelectionFlags`, found integer
    |                |
    |                expected because this is `SelectionFlags`


```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,7 +46,7 @@ futures-intrusive = "0.5.0"
 parking_lot = "0.12"
 bytemuck = { version = "1.12.1", features = ["derive"] }
 smallvec = "1.8.0"
-fello = { git = "https://github.com/dfrg/fount", rev = "8ee330bb9b59291edf2feaa6968ca834a58ae712" }
+fello = { git = "https://github.com/dfrg/fount", rev = "58a284eaae67512fb61cf76177c5d33238d79cb1" }
 peniko = { git = "https://github.com/linebender/peniko", rev = "cafdac9a211a0fb2fec5656bd663d1ac770bcc81" }
 guillotiere = "0.6.2"
 

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,7 +46,7 @@ futures-intrusive = "0.5.0"
 parking_lot = "0.12"
 bytemuck = { version = "1.12.1", features = ["derive"] }
 smallvec = "1.8.0"
-fello = { git = "https://github.com/dfrg/fount", rev = "a8c0686ca9da236420c04d1f476c41cf7fe6d2ba" }
+fello = { git = "https://github.com/dfrg/fount", rev = "8ee330bb9b59291edf2feaa6968ca834a58ae712" }
 peniko = { git = "https://github.com/linebender/peniko", rev = "cafdac9a211a0fb2fec5656bd663d1ac770bcc81" }
 guillotiere = "0.6.2"
 

